### PR TITLE
bump version, update CHANGELOG for v2.0.2

### PR DESCRIPTION
**Pull request by josecorella**
*Issue #, if available:*

*Description of changes:*
Updating CHANGELOG for version bump 2.0.2

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+([#139](https://github.com/aws/aws-dynamodb-encryption-java/pull/139))
+
+Maintenance:
+* chore: bump aws-java-sdk-bom from 1.11.460 to 1.12.51
+ ([#154](https://github.com/aws/aws-dynamodb-encryption-java/pull/154))
+* chore: Add support policy
+([#149](https://github.com/aws/aws-dynamodb-encryption-java/pull/149))
+* CI: Create release builds that validate on multiple JDKs
+([#148](https://github.com/aws/aws-dynamodb-encryption-java/pull/148))
+* chore: Add exmaple for MRKs
```
Comment (farleyb-amazon) [inline CHANGELOG.md:22]:
```suggestion
* chore: Add example for MRKs
```

Comment (josecorella) [reply CHANGELOG.md:22]: sometimes I think I can type and then do stuff like this 🤦‍♂️

**Comment by lavaleri** [comment]
Should this PR also be updating the version in the pom/README?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+## 2.0.2 -- 2021-08-30
+Fixes:
+* NPE if reading record without a signature fields
+([#152](https://github.com/aws/aws-dynamodb-encryption-java/pull/152))
+* Add Formatter
+([#146](https://github.com/aws/aws-dynamodb-encryption-java/pull/146))
+* Update user agent string to correctly reflect version
+([#141](https://github.com/aws/aws-dynamodb-encryption-java/pull/141))
+* Moving Javadoc to webroot
+([#140](https://github.com/aws/aws-dynamodb-encryption-java/pull/140))
```
Comment (lavaleri) [inline CHANGELOG.md:11]: Remove. This was done on the gh-pages branch, so I don't think it belongs here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
 # Changelog
+## 2.0.2 -- 2021-08-30
+Fixes:
```
Comment (lavaleri) [inline CHANGELOG.md:3]: NP
```suggestion
### Fixes
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+* Add Formatter
+([#146](https://github.com/aws/aws-dynamodb-encryption-java/pull/146))
+* Update user agent string to correctly reflect version
+([#141](https://github.com/aws/aws-dynamodb-encryption-java/pull/141))
+* Moving Javadoc to webroot
+([#140](https://github.com/aws/aws-dynamodb-encryption-java/pull/140))
+* Data model migration doc breadcrumb.
+([#139](https://github.com/aws/aws-dynamodb-encryption-java/pull/139))
+
+Maintenance:
```
Comment (lavaleri) [inline CHANGELOG.md:15]: NP
```suggestion
### Maintenance
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
 # Changelog
+## 2.0.2 -- 2021-08-30
+Fixes:
+* NPE if reading record without a signature fields
+([#152](https://github.com/aws/aws-dynamodb-encryption-java/pull/152))
+* Add Formatter
+([#146](https://github.com/aws/aws-dynamodb-encryption-java/pull/146))
```
Comment (lavaleri) [inline CHANGELOG.md:7]: I would move this down to "maintenance".

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,29 @@
 # Changelog
+## 2.0.2 -- 2021-08-30
```
Comment (lavaleri) [inline CHANGELOG.md:2]:
```suggestion
## 2.0.2 -- 2021-08-31
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,29 @@
...
+### Fixes
+* NPE if reading record without a signature fields
```
Comment (lavaleri) [inline CHANGELOG.md:4]:
```suggestion
* fix: Allow reading old plaintext records after adding new encrypted field to schema
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,29 @@
...
+([#152](https://github.com/aws/aws-dynamodb-encryption-java/pull/152))
+* Update user agent string to correctly reflect version
```
Comment (lavaleri) [inline CHANGELOG.md:6]:
```suggestion
* fix: Update user agent string to correctly reflect version
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,4 +1,29 @@
...
+([#141](https://github.com/aws/aws-dynamodb-encryption-java/pull/141))
+* Data model migration doc breadcrumb.
```
Comment (lavaleri) [inline CHANGELOG.md:8]:
```suggestion
* fix: Data model migration doc breadcrumb.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,4 +1,29 @@
...
+([#139](https://github.com/aws/aws-dynamodb-encryption-java/pull/139))
+
+### Maintenance
+* chore: bump aws-java-sdk-bom from 1.11.460 to 1.12.51
+([#154](https://github.com/aws/aws-dynamodb-encryption-java/pull/154))
+* chore: Add support policy
+([#149](https://github.com/aws/aws-dynamodb-encryption-java/pull/149))
+* CI: Create release builds that validate on multiple JDKs
+([#148](https://github.com/aws/aws-dynamodb-encryption-java/pull/148))
+* Add Formatter
```
Comment (lavaleri) [inline CHANGELOG.md:18]:
```suggestion
* chore: Add format check to CI and format
```

**Review by lavaleri — APPROVED** [review]

**Pull request merged**